primitives/message-lane/src/lib.rs

```diff
@@ -126,6 +126,8 @@ impl Default for OutboundLaneData {
 decl_runtime_apis! {
 	/// Outbound message lane API.
 	pub trait OutboundLaneApi {
+		/// Returns dispatch weight of all messages in given inclusive range.
+		fn messages_dispatch_weight(lane: LaneId, begin: MessageNonce, end: MessageNonce) -> Weight;
```

Comment:
So the relayer will have some strategy of bin-searching the right nonce range to make sure it fits in a single transaction, right?

Reply:
Something like that, right